services/QuillLMS/app/queries/progress_reports/district_standards_reports.rb

```diff
@@ -3,8 +3,11 @@
 class ProgressReports::DistrictStandardsReports
   attr_reader :admin_id
```

Comment:
Is `is_fremium_view` missppelled?  Kind of reminds me of fremen.   

I would rename variable to just `is_freemium` and add it to attr_reader here:

`attr_reader :admin_id, :is_freemium`

Then you can get rid of a lot of the parameter passing as follow:

```
def results
  ...
  ids = user_ids
  return [] if ids.empty?

  query(ids)

end

# is_freemium doesn't need to be passed if it's an attr_reader
private def query(user_ids)
  ...
end

# admin_id doesn't need to be passed since it's an attr_reader
private def user_ids
  ...
end
```